cpp/src/slice2java/JavaUtil.cpp

```diff
@@ -1720,6 +1722,16 @@ Slice::JavaGenerator::validateMetadata(const UnitPtr& u)
         .acceptedArgumentKind = MetadataArgumentKind::SingleArgument,
         .extraValidation = [](const MetadataPtr&, const SyntaxTreeBasePtr& p) -> optional<string>
         {
+            // TODO uncomment after replacing 'java:package' with 'java:identifier'.
+            // const string msg = "'java:package' is deprecated; use 'java:identifier' to remap modules instead";
+            // p->unit()->warning(metadata->file(), metadata->line(), Deprecated, msg);
+
+            if (auto cont = dynamic_pointer_cast<Contained>(p); cont && cont->hasMetadata("java:identifier"))
```

Comment:
Please name this variable `contained`.

Reply:
This was just copy-pasted from elsewhere. Instead of `cont` I went with `element`.

So, still no short-hand naming! But `element` is semantically closer to what we're getting here.
Even if the actual type just happens to be `Contained`.